keys/google/google_test.go

```diff
@@ -0,0 +1,38 @@
+package google
+
+import (
+	"context"
+	"testing"
+
+	"github.com/flow-hydraulics/flow-wallet-api/configs"
+	"github.com/flow-hydraulics/flow-wallet-api/keys"
+)
+
+func TestGenerate(t *testing.T) {
+	opts := &configs.Options{EnvFilePath: "../../.env.test"}
+	testCfg, err := configs.ParseConfig(opts)
```

Comment:
I would manually set `testCfg.DatabaseDSN` to something safe here, just in case.

Reply:
The tests shouldn't touch the database here, but better safe than sorry :)